src/jwplayer/players/MobileArticleVideoPlayer/MobileArticleVideoPlayer.tsx

```diff
@@ -61,6 +65,9 @@ export const MobileArticleVideoPlayerContent: React.FC<MobileArticleVideoPlayerP
 
 	const onPlayerInstanceReady = (playerInstance) => {
 		articlePlayerOnReady(videoDetails, playerInstance);
+		communicationService.on('[JWPlayer] Player Ready', async () => {
```

Comment:
Same as above, based on the callback's name, it's being called when the player instance is ready 🤔

Reply:
Yes